docs/pages/blog/introducing-pigment-css.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Introducing Pigment CSS: the next generation of CSS-in-JS for Material UI'
+description: 'Pigment CSS offers significant performance gains along with RSC and App Router support.'
+date: 2024-05-14T00:00:00.000Z
+authors: ['samuelsycamore']
+tags: ['Pigment CSS']
+card: false
+---
+
+In the era of React Server Components and the Next.js App Router, component libraries like Material UI must make some paradigm-shifting changes to reap the potential performance gains by moving more of the work of rendering UIs from run-time to build-time.
```

Comment:
```suggestion
In the era of React Server Components and the Next.js App Router, component libraries like Material UI must make some paradigm-shifting changes to reap the potential performance gains by moving more of the work of rendering UIs from client to server.
```

Reply:
It does not necessarily mean that everything happens at build time. That mainly depends on the way users set the next.config to `export` mode.